cmd/gosec/main.go

```diff
@@ -187,7 +190,7 @@ func loadRules(include, exclude string) rules.RuleList {
 	return rules.Generate(filters...)
 }
 
-func saveOutput(filename, format string, color bool, paths []string, issues []*gosec.Issue, metrics *gosec.Metrics, errors map[string][]gosec.Error) error {
```

Comment:
Nice! What do you think about replacing this parameter addition to the saveOutput method and replacing that with something like this
```go
rootpaths := getRootPaths(paths)
if filename ! = "" {
  saveReport(filename, ...)// this saves the report in a file
}
if filename == "" or stdout {
  printReport(... ) // this prints the report in the stdout 
}
```

Reply:
Absolutely! 
This is more adaptable.
adding the same!!